security/pkg/pki/ca/ca.go

```diff
@@ -210,23 +226,85 @@ func NewSelfSignedIstioCAOptions(ctx context.Context,
 	return caOpts, err
 }
 
-func loadSelfSignedCaSecret(client corev1.CoreV1Interface, namespace string, caCertName string, rootCertFile string, caOpts *IstioCAOptions) error {
+// loadCASecrets loads the original istio-ca-secret Secret from K8S.
+// To simplify - this secret should be used for both self-generated and user defined intermediary certs.
+// TODO: rename to loadRootsFromSecret.
+func loadCASecrets(client corev1.CoreV1Interface, namespace string, caCertName string, rootCertFile string, caOpts *IstioCAOptions) error {
```

Comment:
Why are we removing logging from this method? I think having this logging is incredibly useful outside of debugging. The root cert is only created or loaded at start up so this doesn't introduce significant logging load for the user. For such a critical component I think having verbose logging is useful.

Reply:
I think all (uncommon) code paths are covered by a log -  this felt redundant. I added more detailed logs showing the Issuer/Subject for the chains - this one is the most common/boring part.